.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Mimir maintainers
+    url: https://github.com/grafana/mimir/blob/main/MAINTAINERS.md
+    about: Please ask the Mimir maintainers listed here.
```

Comment:
Do we want to encourage people to contact maintainers directly?

Reply:
I'm also skeptical about this. It could lead to spamming of individuals.

Reply:
I have now removed that part of the config.